nexus/bin/eshdf

```diff
@@ -297,6 +315,28 @@ def kinetic():
             #end for
         #end for
     #end if
+
+    if opt.fnk3d is not None:
+        ispin0 = int(opt.ispin)
```

Comment:
Move contents to new function (e.g. `write_nk`).  The `--fnk3d` option could simply be something like `-o` and `--outfile` (not a big deal though).